src/mcts/params.cc

```diff
@@ -360,18 +360,10 @@ const OptionId SearchParams::kMaxConcurrentSearchersId{
     "max-concurrent-searchers", "MaxConcurrentSearchers",
     "If not 0, at most this many search workers can be gathering minibatches "
     "at once."};
-const OptionId SearchParams::kDrawScoreSidetomoveId{
-    "draw-score-sidetomove", "DrawScoreSideToMove",
-    "Score of a drawn game, as seen by a player making the move."};
-const OptionId SearchParams::kDrawScoreOpponentId{
-    "draw-score-opponent", "DrawScoreOpponent",
-    "Score of a drawn game, as seen by the opponent."};
-const OptionId SearchParams::kDrawScoreWhiteId{
-    "draw-score-white", "DrawScoreWhite",
-    "Adjustment, added to a draw score of a white player."};
-const OptionId SearchParams::kDrawScoreBlackId{
-    "draw-score-black", "DrawScoreBlack",
-    "Adjustment, added to a draw score of a black player."};
+const OptionId SearchParams::kDrawScoreId{
+    "draw-score", "DrawScore",
+    "Adjustment of the draw score from white's perspective. Value 0 gives "
+    "standard scoring, value -1 gives Armageddon scoring."};
```

Comment:
If there's going to be "value X for Armageddon" type examples here, should https://github.com/LeelaChessZero/lc0/blob/78155977f745aafac7042a4ac7797096dd2a2656/src/mcts/params.cc#L397-L401 get something similar?

Reply:
In general I'd prefer to not mix the different PR topics.

About your question: There will be a longer blog post for the next release, explaining the features, the parameters and their meaning in much more detail than a few lines of code (which people using a GUI won't even see) could possibly transport.